website/cue/reference/remap/functions/get_metadata_field.cue

```diff
@@ -0,0 +1,38 @@
+package metadata
+
+remap: functions: get_metadata_field: {
+	category: "System"
+	description: """
+		Returns the value of the given field from the event metadata.
+		"""
+
+	arguments: [
+		{
+			name: "key"
+			description: '''
+				   The name of the field to look up in the metadata.
+				'''
+			required: true
+			enum: {
+				"datadog_api_key": '''
+					   The Datadog API key.
+					   This exists if the `store_api_key` setting is true in the `datadog_agent` source.
```

Comment:
```suggestion
					   The Datadog API key.
					   
					   This exists if the `store_api_key` setting is true in the `datadog_agent` source.
```

I think the extra line break is necessary if you actually want a line break in the output since wrapped text is rendered as one flowing "line".